compiler-rt build fails with several warnings and syntax errors

Trying to build compiler-rt under openSUSE Tumbleweed with clang13 (it's the default version now).
The osxcross toolchain builds fine, however compiler-rt throws several warnings and syntax errors like this:

```
osxcross: warning: possibly dangerous include path specified: '-I /usr/include'
...
In file included from /home/dmitry/devel/osxcross/build/compiler-rt/compiler-rt/lib/orc/macho_platform.cpp:13:
In file included from /home/dmitry/devel/osxcross/build/compiler-rt/compiler-rt/lib/orc/macho_platform.h:16:
In file included from /home/dmitry/devel/osxcross/build/compiler-rt/compiler-rt/lib/orc/common.h:16:
In file included from /home/dmitry/devel/osxcross/build/compiler-rt/compiler-rt/lib/orc/c_api.h:22:
/usr/include/stdlib.h:102:32: error: expected ';' after top level declarator
     __THROW __attribute_pure__ __nonnull ((1)) __wur;
                               ^
                               ;
```

--- a/build_compiler_rt.sh
+++ b/build_compiler_rt.sh
@@ -127,6 +127,9 @@ if [ $f_res -eq 1 ]; then
     $SED -i 's/${CMAKE_COMMAND} -E ${COMPILER_RT_LINK_OR_COPY}/ln -sf/g' \
       lib/builtins/CMakeLists.txt
 
+    $SED -i 's/list(APPEND ORC_CFLAGS -I${DIR})//g' \
+      lib/orc/CMakeLists.txt
+
     if [ $HAVE_OS_LOCK -eq 0 ]; then
       $SED -i "s/COMPILER_RT_HAS_TSAN TRUE/COMPILER_RT_HAS_TSAN FALSE/g" \
         cmake/config-ix.cmake

